src/azure-cli/azure/cli/command_modules/cosmosdb/_params.py

```diff
@@ -128,17 +132,20 @@ def load_arguments(self, _):
         c.argument('account_name', account_name_type, id_part=None)
         c.argument('database_name', options_list=['--name', '-n'], help="Database name")
         c.argument('throughput', help='The throughput of SQL database (RU/s). Default value is 400')
+        c.argument('max_throughput', max_throughput_type)
 
 # SQL container
     with self.argument_context('cosmosdb sql container') as c:
         c.argument('account_name', account_name_type, id_part=None)
         c.argument('database_name', database_name_type)
         c.argument('container_name', options_list=['--name', '-n'], help="Container name")
         c.argument('partition_key_path', options_list=['--partition-key-path', '-p'], help='Partition Key Path, e.g., \'/address/zipcode\'')
+        c.argument('partition_key_version', type=int, options_list=['--partition-key-version'], help='Indicates the version of partition key.')
```

Comment:
active voice will be exprected. `Indtictates` -> `Indicated`

Reply:
Would do.